### ISSUE TITLE
i am running your code now with your run.sh (change batch_size from 128 to 32), i could get 92.7 at 37 epochs. But after 50 epochs , it start descend slowly(at epoch 85 i get 82.3).  what should i do?

### ISSUE THREAD

**Issue by bili1bili** [YabinZhang1994/CDAN-re-implement#1]

**Comment by YBZh** [comment]
1. Maybe early stopping can help.
2. Fine-tuning the learning rate schedule or domain weight schedule according to your experiment setting should be helpful. 

Overfitting also occurs in my experiments, but it is not so serious: in the A2W task, the best performance occurs at ~160 epoch (93.6%), and it degenerates to ~90% at the end of training (200 epoch).

**Comment by bili1bili** [comment]
thanks for your reply. i find it could get a nice performance (90.3%) at the end of training.
